ansible/roles/ocp-workload-starter-workshop/defaults/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+# PROVIDED BY THE INFRA
+# Uncomment and set a value to take effect. Else, will use the defaults
+#
+# ocp_apps_domain:
+admin_project: 
+user_count_ini: 1
+user_count_end: 50
+user_format: user%02d
+
+##
+# VALUES SPECIFIC TO THIS WORKLOAD
+# Uncomment and set a value to take effect. Else, will use the defaults
+#
+user_gogs_admin: "gogsadmin"
+user_gogs_password: "r3dh4t1!"
```

Comment:
please have this in env_secret_vars.yml (not versionned) instead and an example using `-e user_gogs_password=...` in readme

Reply:
where would env_secret_vars.yml go? Is this file picked automatically like defaults/main.yml?